scripts/generateDepcheckrc.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2023 The Jaeger Authors.
```

Comment:
so is this file needed or is it replaced by the two subpackage-specific files below?

Reply:
It's now replaces by two subpackage specific files now.